<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-418NQCC" ignored="7" name="Test(failed)(failed)(failed)(failed)(failed)" tests="8" failures="3" timestamp="21 Nov 2022 10:08:13 GMT" time="25.414" errors="0">
  <testcase name="Test4" time="1.657" classname="org.test.saucedemo.SauceDemo">
    <skipped/>
  </testcase> <!-- Test4 -->
  <testcase name="Test4" time="1.201" classname="org.test.saucedemo.SauceDemo">
    <skipped/>
  </testcase> <!-- Test4 -->
  <testcase name="Test4" time="1.324" classname="org.test.saucedemo.SauceDemo">
    <skipped/>
  </testcase> <!-- Test4 -->
  <testcase name="Test4" time="0.924" classname="org.test.saucedemo.SauceDemo">
    <skipped/>
  </testcase> <!-- Test4 -->
  <testcase name="Test4" time="0.9" classname="org.test.saucedemo.SauceDemo">
    <skipped/>
  </testcase> <!-- Test4 -->
  <testcase name="Test4" time="0.913" classname="org.test.saucedemo.SauceDemo">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [manikandan] but found [secret_sauce]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [manikandan] but found [secret_sauce]
at org.test.saucedemo.SauceDemo.Test4(SauceDemo.java:101)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- Test4 -->
  <testcase name="Test4" time="0.905" classname="org.test.saucedemo.SauceDemo">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [manikandan] but found [secret_sauce]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [manikandan] but found [secret_sauce]
at org.test.saucedemo.SauceDemo.Test4(SauceDemo.java:101)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- Test4 -->
  <testcase name="Test4" time="0.867" classname="org.test.saucedemo.SauceDemo">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [manikandan] but found [secret_sauce]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [manikandan] but found [secret_sauce]
at org.test.saucedemo.SauceDemo.Test4(SauceDemo.java:101)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- Test4 -->
  <testcase name="Test5" time="0.0" classname="org.test.saucedemo.SauceDemo">
    <ignored/>
  </testcase> <!-- Test5 -->
  <testcase name="Test6" time="0.0" classname="org.test.saucedemo.SauceDemo">
    <ignored/>
  </testcase> <!-- Test6 -->
  <testcase name="Test3" time="0.0" classname="org.test.saucedemo.SauceDemo">
    <ignored/>
  </testcase> <!-- Test3 -->
  <testcase name="Test1" time="0.0" classname="org.test.saucedemo.SauceDemo">
    <ignored/>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.0" classname="org.test.saucedemo.SauceDemo">
    <ignored/>
  </testcase> <!-- Test2 -->
  <testcase name="datapro" time="0.0" classname="org.test.saucedemo.SauceDemo">
    <ignored/>
  </testcase> <!-- datapro -->
  <testcase name="datapro1" time="0.0" classname="org.test.saucedemo.SauceDemo">
    <ignored/>
  </testcase> <!-- datapro1 -->
</testsuite> <!-- Test(failed)(failed)(failed)(failed)(failed) -->
